### PR TITLE
Migrate /policies/participation/ to fluent. (#9666) [skip l10n]

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Migrates /about/governance/policies/participation/ to fluent.

## Issue / Bugzilla link
#9666 

## Testing
http://localhost:8000/en-US/about/governance/policies/participation/

### CODE REVIEW

**Comment by amychurchwell** [comment]
@alexgibson would you be able to take a look at this for me? I keep getting errors locally and I am not sure where the problem is. I have ran `make clean build` and have also started over with the same results.

File: bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html [hunk 1]
```diff
@@ -4,604 +4,294 @@
...
   <ul class="mzp-u-list-styled">
-    <li>{{ _('Additional Resources: <a href="%s">How to Report</a>')|format(url('mozorg.about.governance.policies.reporting')) }}</li>
-  {% if l10n_has_tag('participation_jan2020') %}
-    <li>{{ _('Questions: <a href="%s">cpg-questions@mozilla.com</a>')|format('mailto:cpg-questions@mozilla.com') }}</li>
-  {% else %}
-    <li>{{ _('Questions: <a href="%s">inclusion@mozilla.com</a>')|format('mailto:inclusion@mozilla.com') }}</li>
-  {% endif %}
+    <li>{{ ftl('participation-additional-resources', url('mozorg.about.governance.policies.reporting')) }}</li>
+
+    <li>{{ ftl('participation-questions-cpg-questionsmozillacom')|format('mailto:cpg-questions@mozilla.com') }}</li>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html:235]:
```suggestion
    <li>{{ ftl('participation-questions-cpg-questionsmozillacom', url='mailto:cpg-questions@mozilla.com') }}</li>
```

File: bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html [hunk 2]
```diff
@@ -4,604 +4,294 @@
...
+  <h2>{{ ftl('participation-modifications-to-these') }}</h2>
+
+  <p>
+  {{ ftl('participation-mozilla-may-amend', guidelines=url('mozorg.about.governance.policies.participation')) }}
+  </p>
 </section>
 
 <footer>
-  <p id="note-1">{{ _('[1] The anti-harassment policy is accessible to paid staff <a href="%s">here</a>.')|format('https://mana.mozilla.org/wiki/display/PR/Open+Door+Policy') }}</p>
+  <p id="note-1">{{ ftl('participation-1-the-anti-harassment')|format('https://mana.mozilla.org/wiki/display/PR/Open+Door+Policy') }}</p>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html:295]:
```suggestion
  <p id="note-1">{{ ftl('participation-1-the-anti-harassment', url='https://mana.mozilla.org/wiki/display/PR/Open+Door+Policy') }}</p>
```

File: l10n/en/mozorg/about/governance/policies/participation.ftl [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+participation-reporting = Reporting
+participation-if-you-believe-youre = If you believe you’re experiencing unacceptable behavior that will not be tolerated as outlined above, <a href="{ $hotline }">please use our hotline to report</a>. Reports go directly to { -brand-name-mozilla }’s Employment Counsel and HR People Partners and are triaged by the Community Participation Guidelines Response Lead.
+participation-after-receiving-a = After receiving a concise description of your situation, they will review and determine next steps. In addition to conducting any investigation, they can provide a range of resources, from a private consultation to other community resources. They will involve other colleagues or outside specialists (such as legal counsel), as needed to appropriately address each situation.
+participation-additional-resources = Additional Resources: <a href="{ $missing-var }">How to Report</a>
+participation-questions-cpg-questionsmozillacom = Questions: <a href="{ $missing-var }">cpg-questions@mozilla.com</a>
```
Comment (alexgibson) [inline l10n/en/mozorg/about/governance/policies/participation.ftl:92]:
```suggestion
participation-additional-resources = Additional Resources: <a href="{ $url }">How to Report</a>
participation-questions-cpg-questionsmozillacom = Questions: <a href="{ $url }">cpg-questions@mozilla.com</a>
```

File: l10n/en/mozorg/about/governance/policies/participation.ftl [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+participation-modifications-to-these = Modifications to these guidelines
+participation-mozilla-may-amend = { -brand-name-mozilla } may amend the guidelines from time to time and may also vary the procedures it sets out where appropriate in a particular case. Your agreement to comply with the guidelines will be deemed agreement to any changes to it. This policy does not form part of any { -brand-name-mozilla } employee’s contract of employment or otherwise have contractual effect.
+participation-1-the-anti-harassment = [1] The anti-harassment policy is accessible to paid staff <a href="{ $missing-var }">here</a>.
```
Comment (alexgibson) [inline l10n/en/mozorg/about/governance/policies/participation.ftl:107]:
```suggestion
participation-1-the-anti-harassment = [1] The anti-harassment policy is accessible to paid staff <a href="{ $url }">here</a>.
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
Looks like the issue is because of a few missing variables than need passing to thir related strings. The changes below seem to fix things for me.

**Comment by amychurchwell** [comment]
migration for the existing translations in progress. en is in review.

**Comment by amychurchwell** [comment]
migrations have merged! https://github.com/mozilla-l10n/www-l10n/pull/138

File: bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html [hunk 5]
```diff
@@ -4,604 +4,294 @@
...
   </p>
 
   <ul class="mzp-u-list-styled">
-    <li>{{ _('Additional Resources: <a href="%s">How to Report</a>')|format(url('mozorg.about.governance.policies.reporting')) }}</li>
-  {% if l10n_has_tag('participation_jan2020') %}
-    <li>{{ _('Questions: <a href="%s">cpg-questions@mozilla.com</a>')|format('mailto:cpg-questions@mozilla.com') }}</li>
-  {% else %}
-    <li>{{ _('Questions: <a href="%s">inclusion@mozilla.com</a>')|format('mailto:inclusion@mozilla.com') }}</li>
-  {% endif %}
+    <li>{{ ftl('participation-additional-resources', url('mozorg.about.governance.policies.reporting')) }}</li>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html:233]:
```suggestion
    <li>{{ ftl('participation-additional-resources', url=url('mozorg.about.governance.policies.reporting')) }}</li>
```

File: bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html [hunk 6]
```diff
@@ -4,604 +4,294 @@
...
-    you wish you receive notice when we update the guidelines, drop
-    us a line and we will add you to the updates notification list.
-  {% endtrans %}
-  </p>
-{% endif %}
+  <h2>{{ ftl('participation-ask-questions') }}</h2>
+
+  <p>
+  {{ ftl('participation-everyone-is-encouraged', mailto_questions='mailto:cpg-questions@mozilla.com',
+           changelog='https://github.com/mozilla/cpg/blob/master/CHANGELOG.md') }}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html:266]: Nit: indentation

File: bedrock/mozorg/urls.py [hunk 7]
```diff
@@ -49,7 +49,9 @@
     page('about/governance/policies/security-group/certs/policy',
          'mozorg/about/governance/policies/security/certs/policy.html'),
     page('about/governance/organizations', 'mozorg/about/governance/organizations.html'),
-    page('about/governance/policies/participation', 'mozorg/about/governance/policies/participation.html'),
+    page('about/governance/policies/participation',
+         'mozorg/about/governance/policies/participation.html',
+         ftl_files=['mozorg/about/governance/policies/participation.ftl']),
```
Comment (alexgibson) [inline bedrock/mozorg/urls.py:54]: Nit: think you can omit the `.ftl` extension here?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Spotted one missing variable, but otherwise this looks good to me?

Do we need to specify a config for these Fluent files? As it stands this would go out to all locales.

**Comment by alexgibson** [comment]
Ok, I saw you added the Fluent config in https://github.com/mozilla/bedrock/pull/9708

Once you rebase / update this PR then I think this should be good to merge as well :)

**Review by alexgibson — APPROVED** [review]
🍡

**Pull request merged**